modules/gateways/coingate.php

```diff
@@ -1,80 +1,154 @@
 <?php
 
-require_once(dirname(__FILE__) . '/CoinGate/init.php');
-require_once(dirname(__FILE__) . '/CoinGate/version.php');
+if (! defined('WHMCS')) {
+    die('This file cannot be accessed directly');
+}
+
+require_once __DIR__ . '/coingate/client.php';
+
+/**
+ * Define module related meta data.
+ *
+ * Values returned here are used to determine module related capabilities and
+ * settings.
+ *
+ * @see https://developers.whmcs.com/payment-gateways/meta-data-params/
+ *
+ * @return array
+ */
+function coingate_MetaData()
+{
+    return [
+        'DisplayName' => 'CoinGate Payment Gateway Module',
+        'APIVersion' => '1.1', // Use API Version 1.1
```

Comment:
APIVersion is 1.1 by default since WHMCS 5.2 (released in 2013). This could be removed or left like this. It's just what their documentation and templates suggested to have. [metadata parameters](https://developers.whmcs.com/domain-registrars/metadata-params/)